### DISCUSSION TITLE
How many collaborators can be added in app.qfield.cloud

### DISCUSSION THREAD

**Discussion by MonikaShah** [opengisch/QField#3204] [Q&A]
I would like to know the count of collaborators that can be added in 
1. Free service
2. Pro Service.

**Comment by suricactus** [comment]
While we are in BETA and this might change a bit, the current idea and implementation is:
|plan\project|public|private|
|-|-|-|
|free|unlimited|0|
|pro|unlimited|1 PRO|
|organization|unlimited|unlimited|

**Reply by suricactus** [reply]
Related question https://github.com/opengisch/QField/discussions/3206

**Reply by QGISter** [reply]
So if I have an organization at the moment in beta and if I change to Pro I will only be able to add one other Pro member and not the staff of my organization?
